src/grocker/resources/docker/compiler-image/compile.py

```diff
@@ -104,9 +104,14 @@ def main():
             "grocker could not become the owner of %s, check the sudoers file", WHEELS_DIRECTORY,
         )
 
-    for release in args.release:
-        if not build_wheels(venv, release, WHEELS_DIRECTORY, args.pip_constraint):
-            exit(1)
+    constraints = os.environ.get('PIP_CONSTRAINT_CONTENT', base64.b64encode(zlib.compress(b'')))
```

Comment:
In python 3, `base64.b64encode` returns `bytes` and os.get returns `str`.
In python 2, `base64.b64encode` returns `str` and os.get returns `str`.

👌